repos/system_upgrade/el7toel8/libraries/config/tests/test_getenvvars.py

```diff
@@ -45,19 +31,19 @@ def test_get_product_type_valid(monkeypatch):
                   'LEAPP_DEVEL_TARGET_PRODUCT_TYPE': dst}
         exp_src = 'ga' if not src else src.lower()
         exp_dst = 'ga' if not dst else dst.lower()
-        monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars))
+        monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars=envars))
         assert exp_src == get_product_type('source')
         assert exp_dst == get_product_type('target')
     # return 'ga' if envars are not specified
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked())
+    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked)
```

Comment:
the same like https://github.com/oamg/leapp-repository/pull/489/files#r414407185